lib/Core/PTreeWriter.cpp

```diff
@@ -0,0 +1,169 @@
+//===-- PTreeWriter.cpp ---------------------------------------------------===//
+//
+//                     The KLEE Symbolic Virtual Machine
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#include "PTreeWriter.h"
+
+#include "PTree.h"
+#include "klee/Support/ErrorHandling.h"
+
+using namespace klee;
+
+
+void prepare_statement(sqlite3 * db, std::string &query, sqlite3_stmt ** stmt) {
+#ifdef SQLITE_PREPARE_PERSISTENT
```

Comment:
Have you ever encountered this being false? I imagine KLEE would not be compiled with stripped down sqlite.

Reply:
Fails in Travis for that crappy OS from Cupertino.

Reply:
Ah brilliant ... They updated the fruity OS in travis now, so maybe its fixed, but I guess it does mean there are uses of this in the wild. We should distribute our own sqlite :p